assembly/pentaho-server/src/main/descriptors/assembly.xml

```diff
@@ -7,48 +7,48 @@
   <baseDirectory>pentaho-server</baseDirectory>
   <files>
     <file>
-      <source>${project.root.build.directory}/${project.artifactId}/pentaho-solutions/system/dialects/hsqldb/applicationContext-spring-security-hibernate.properties</source>
+      <source>${project.root.build.directory}/pentaho-server/pentaho-solutions/system/dialects/hsqldb/applicationContext-spring-security-hibernate.properties</source>
```

Comment:
http://jira.pentaho.com/browse/BACKLOG-10262

One of the requirements of the work of merging BA Server and DI Server into a single Pentaho Server was also that we would no longer get a differentiation between CE and EE distributables embedded in the folder-name. To sum it up, the folder is meant to be named "pentaho-server" ( regardless of it being CE or EE )  

Doing this would mean that the unzipped artifact would have a folder structure of <unzip-dir>/pentaho-server*-ce*/pentaho-solutions/(...)